### DISCUSSION TITLE
I'm having trouble with the styles in the modal gallery

### DISCUSSION THREAD

**Discussion by whoisnegrello** [awcodes/filament-curator#440] [Q&A] [answered]
I don't know what the problem is, but the curator.css styles are not included when I use CuratorPicker in a Fabricator block.
![image](https://github.com/awcodes/filament-curator/assets/11581735/07e60f5c-5a4d-4dd8-902b-2cc2ee73df44)
All items are full width because the grid styles are not applied.

**Comment by awcodes** [comment]
I can't speak to any inclusion in another plugin. But as long as your theme is setup correctly and you followed the installation instructions. It should be fine.

**Reply by whoisnegrello** [reply]
I tried it outside the plugin, using only Filament and Curator and I have the same problem.
I followed all the steps but it seems that it is not loading the public/css/awcodes/curator/curator.css file.
Any suggestions on how to incorporate them?
Thanks for your reply.

**Reply by awcodes** [reply, marked as answer]
It shouldn't be loading the stylesheet, you should be @import the styles into your theme as per the read me.

**Reply by whoisnegrello** [reply]
Thank you very much for your answers, I was able to do it, I needed to incorporate the viteTheme into the panel.
![Screen Shot 2024-02-16 at 22 18 58](https://github.com/awcodes/filament-curator/assets/11581735/044147af-4488-429e-aa0b-3254aeab9645)
![image](https://github.com/awcodes/filament-curator/assets/11581735/d5a09e17-b9db-480e-b631-577fd28e197f)
Now it works perfectly!

**Reply by MarkGersaliaPH** [reply]
Hi Where did you put the import? its seems its not working with me, my project doesn't have a tailwind.config.js

**Reply by beelalamin** [reply]
As per the [docs:](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)

⇂ First, add a new item to the `input` array of `vite.config.js`: `resources/css/filament/admin/theme.css`
⇂ Next, register the theme in the admin panel provider using `->viteTheme('resources/css/filament/admin/theme.css')`
⇂ Finally, run `npm run build` to compile the theme
